ansible/manager-part-0.yml

```diff
@@ -5,13 +5,15 @@
 
   vars:
     repo_path: /home/ubuntu/src/github.com
+    lock_timeout: 300
```

Comment:
Let's use `apt_lock_timeout` as variable name, that is more specific and can also be used in the collections later.

Reply:
Done